earth_enterprise/src/fusion/autoingest/plugins.src/PacketGen.src

```diff
@@ -85,9 +88,11 @@ PacketGenFactory::ReuseOrMakeAndUpdateSubAsset(
 
 template <typename ProductAssetVersion>
 void PacketGenAssetVersionImplD::UpdateChildren(
-    const PacketGenExtraUpdateArg<ProductAssetVersion> &extra) {
+    const PacketGenExtraUpdateArg<ProductAssetVersion> &extra,
+    bool skip, const std::vector<uint>& _neededIndexes) {
```

Comment:
I feel like `skip` may be ambiguous for anyone looking at the function signature. Suggest renaming here and other uses to something that makes it clear what is being skipped.

Reply:
this is getting removed